drivers/flash/Kconfig.sc

```diff
@@ -0,0 +1,36 @@
+# Copyright (c) 2025 Space Cubics, Inc.
+# SPDX-License-Identifier: Apache-2.0
+
+menuconfig SC_QSPI_NOR
+	bool "Space Cubics QSPI driver"
+	default y
+	depends on DT_HAS_SC_QSPI_NOR_ENABLED
+	select FLASH_HAS_DRIVER_ENABLED
+	select FLASH_HAS_EXPLICIT_ERASE
+	select FLASH_HAS_PAGE_LAYOUT
+	help
+	  Enable support for Space Cubics NOR QSPI driver.
```

Comment:
"NOR QSPI driver"
It seems that the terminology is inconsistent in several places.